B827EB4D73D3EFEF.json

```diff
@@ -0,0 +1,19 @@
+{
+    "gateway_config": {
+        "gateway_ID": "B827EB4D73D3EFEF",
+        "servers":[
+            {
+                "server_address": "router.eu.thethings.network",
```

Comment:
This URL should be updated. This is the old v2 routers, please use the v3 ones and then we can merge.